cluster/setup.go

```diff
@@ -317,6 +351,7 @@ func AddM3Admin(name, email string) error {
 		fmt.Println("Error adding user:", err.Error())
 		return err
 	}
+	apptCtx.Commit()
```

Comment:
you already do commit inside `AddAdminAction`
also check `AddAdminAction` cause we never do Rollback